### DISCUSSION TITLE
My Pc is loud like a helicopter

### DISCUSSION THREAD

**Discussion by cl86of** [jef/streetmerchant#1963] [Help]
Hi guys why my pc is loud like a helicopter after 1 min streetmerchant? 
i use :LOW_BANDWIDTH=true 
but its not better

**Comment by viper-666** [comment]
My laptop too...
I also uses LOW_BANDWIDTH="true"

**Comment by outfielder** [comment]
The problem is that street merchant is opening a lot of instances in chromium and as a result your processor is tasked heavily, which causes it to overheat, which causes your computer fans to ramp up.

Try adding longer delays between the searches or narrowing down the items you search for.

**Comment by cousinoctavia** [comment]
Yeah, agree with this comment, it would be great to have a way to throttle the activity of the app to make it slightly more sensible a thing to leave running in the background.
